modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -40,6 +39,32 @@ For more information about features, see xref:identity-management.adoc[{keycloak
 
 . To demonstrate {unifiedpush-service}, xref:push-notifications.adoc#provisioning-push-notifications-service[Provision the {unifiedpush-service} service]. For more information about features, see xref:push-notifications.adoc[{unifiedpush-service} Service]
 
+. To demonstrate {sync-service}:
```

Comment:
NOTE: You can only demonstrate {sync-service} on Cordova.

Reply:
I mentioned it in the features section, should I mention it here too?

Reply:
I think it's worth it, in case someone jumps into docs at this point

Reply:
ok